SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -317,6 +320,41 @@ protected void _userUpdate(Map<String, String> attributes) {
         queueEvent("user", parameters, null);
     }
 
+    protected void _userUpdateDate (Map<String, Date> attributes) {
```

Comment:
~~~ if we are only going to have one key we could remove the for and move this logic into the other _userUpdateDate